evaluator/src/main/resources/rules/osadl.rules.kts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+// Use this rule like:
+//
+// $ ort evaluate -i scanner/src/funTest/assets/dummy-expected-output-for-analyzer-result.yml --rules-resource /rules/osadl.rules.kts
+
+import org.ossreviewtoolkit.evaluator.osadl.Compatibility
+import org.ossreviewtoolkit.evaluator.osadl.matrix
+
+val ruleSet = ruleSet(ortResult, licenseInfoResolver) {
+    val licenseView = LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED
+
+    // Define a rule that is executed for each dependency.
+    dependencyRule("OSADL compliance matrix") {
+        // Requirements for the rule to trigger a violation.
+        require {
+            -isExcluded()
+        }
+
+        val projectLicenseInfo = licenseInfoResolver.resolveLicenseInfo(project.id).filter(licenseView)
+        val outboundLicenses = projectLicenseInfo.licenses.map { it.license }
+
+        // Define a rule that is executed for each license of the dependency.
+        licenseRule("OSADL project license compatibility", licenseView) {
+            outboundLicenses.forEach { outboundLicense ->
+                val compatibilityInfo = matrix
+                    // Be conservative and use the simple license string without the exception string for lookup.
+                    // Eventually resulting false-positives are better than an incompatibility to be missed.
```

Comment:
Do you mean "Potentially"?

Reply:
No, I indeed mean "eventually" in the German sense of "schließlich".